src/server/pfs/cmds/cmds_test.go

```diff
@@ -513,6 +513,7 @@ func TestCmdListRepo(t *testing.T) {
 }
 
 func TestBranchNotFound(t *testing.T) {
+	// TODO(albert) come back and fix the match expression
```

Comment:
I think this comment was added by mistake. We are matching on the correct error message here. The error message comes from `pfs.ErrBranchNotFound` instead of `pfsdb.ErrBranchNotFound`.